ledger/src/snapshot_utils.rs

```diff
@@ -151,6 +151,7 @@ where
 }
 
 pub fn add_snapshot<P: AsRef<Path>>(snapshot_path: P, bank: &Bank) -> Result<()> {
+    trace!("ryoqun saving snapshot");
     bank.purge_zero_lamport_accounts();
```

Comment:
Note: We can reduce the frequency of `purge_zero_lamport_accounts` invocations for the performance gains because the invocation is completely optional for creating snapshots other than freeing up some memory consumed zero-lamport accounts.